recipe/meta.yaml

```
@@ -11,30 +11,47 @@ source:
   sha256: "{{ sha256 }}"
 
 build:
+  # No mongodb on s390x
+  skip: True  # [py<34 or s390x]
   number: 0
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv --no-build-isolation
```

```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-build-isolation -vv
```
Less verbosity would be better for our CI

---

`boo! --verbose` 😆